src/globals.js

```diff
@@ -40,7 +40,7 @@ function joiner(sep) {
 function globals() {
     return {
         range: function(start, stop, step) {
-            if(!stop) {
+            if(!stop && stop !== 0) {
```

Comment:
Sure thing, I'll add the requested change then. My main concern was breaking backwards compatibility as little as possible.